### PR TITLE
fix(promicro_pretty.js): Replace p.xy with p.eaxy due to update of ergogen

### PR DESCRIPTION
**Pull request by ascodeasice**
Hi, when I was running the ergogen in this project, there seem to be an error which says `p.xy` is not a function, and after some digging, I found that it was because ergogen had an update from ergogen in this [commit](https://github.com/ergogen/ergogen/commit/1f57ec4e190785b0a02576b7aac7cafbe5d0fb34), which replaces the `xy` function with `eaxy` function (or esxy function, I might be wrong), thus causing the error. So I've updated the function and ran the command, which has no error this time.

Tldr: Due to ergogen's update, the function in `promicro_pretty.js` is deprecated, and I replaced it.

Btw, I was trying to run the `ergogen` command so that I can get a kicad file such that I can modify the silk layer, but the result doesn't seem right(there is no route in the generated file), can you tell me how to fix it?

### CODE REVIEW

**Comment by johnlamb** [comment]
Thanks for fixing this. Works for me with newest commit, great!
It is correct that there is no route in the kicad file. To keep the ergogen file as clean as possible I have opted to not add in the routes after the generation (I do not know of a good way to auto route them). 
The routing has to be done manually and to your preference.
You can check my generated (and routed) files either in the gerber or ergogen/output/pcbs.
